app/views/users/sessions/new.html.erb

```diff
@@ -0,0 +1,26 @@
+<h2>Log in</h2>
+
+<%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password %><br />
+    <%= f.password_field :password, autocomplete: "current-password" %>
+  </div>
+
+  <% if devise_mapping.rememberable? %>
+    <div class="field">
+      <%= f.check_box :remember_me %>
+      <%= f.label :remember_me %>
+    </div>
+  <% end %>
+
+  <div class="actions">
+    <%= f.submit "Log in" %>
+  </div>
+<% end %>
```

Comment:
- [ ] Good job so far, users can log in and signup as well. I think it would be better if you style your login page as you did for the signup page 🙃 😉 

Your design

<img width="441" alt="Screenshot 2023-04-12 at 22 10 15" src="https://user-images.githubusercontent.com/74501523/231562011-e63947a9-0daf-4dc9-9712-5054eb523cee.png">

Original design

<img width="441" alt="Screenshot 2023-04-12 at 22 10 50" src="https://user-images.githubusercontent.com/74501523/231562097-32c24b3d-75b3-4ad5-b54b-6a73881b3f64.png">